keyboards/gorthage_truck/info.json

```diff
@@ -1,35 +1,209 @@
 {
     "keyboard_name": "gorthage_truck",
     "url": "https://github.com/jpuerto96",
-    "maintainer": "s8erdude",
-    "width": 17,
-    "height": 4,
+    "maintainer": "jpuerto96",
```

Comment:
Please change this to s8erdude.

Reply:
How about using `jpuerto96 (s8erdude)` here? Per the QMK Docs, this field is meant to be the GitHub username of the keyboard maintainer.

[info.json Format - QMK Firmware Documentation](https://docs.qmk.fm/#/reference_info_json?id=infojson-format)

Reply:
That would be fine w me.